Europa/Schweiz/morges/morges.md

```diff
@@ -0,0 +1,40 @@
+
+## Geschichte
+
+Noch im 13. Jahrhundert bestand der Ort nur aus ein paar Häusern am Seeufer. Im Jahr 1286 begann unter Ludwig I. von Savoyen der Bau des Schlosses. Die Gründung des Marktfleckens durch die Savoyer war gegen die Machtausweitung der Bischöfe von Lausanne gerichtet. Ferner wollten die Savoyer damit den Handelsweg zwischen Lausanne und Genf kontrollieren und eine neue Station der Handelsschifffahrt auf dem Genfersee gründen.  
+
+Die erste urkundliche Erwähnung erfolgte 1297 tempore quo Morgia constructa est (in der Zeit, als Morges gegründet wurde). Nach älterer, auf Johann Ulrich Hubschmied zurückgehender Ansicht leitet sich der Ortsname von einem hypothetischen keltischen *morga «Grenze» ab; die heute angenommene Deutung von Theodora Geiger, Bruno Boesch und Wulf Müller führt den Namen auf indogermanisch *mer(e)g- u. ä. «Sumpf, sumpfiger Fluss» zurück. Der Name des Ortes ist damit sekundär vom Namen des Gewässers abgeleitet.[3]
+
+Der nordöstlich des Schlosses entstandene Marktflecken besass seit seiner Gründung das Stadtrecht. Morges war von 1359 bis 1536 zeitweise Residenz der Grafen und Herzöge von Savoyen.  
+
+Mit der Eroberung der Waadt durch Bern im Jahr 1536 wurde Morges Hauptort und das Schloss Amtssitz der gleichnamigen Vogtei. Die Vogtei Morges umfasste in der Berner Zeit neben dem Gebiet des heutigen Bezirks Morges auch die Gebiete der Bezirke Cossonay und Rolle sowie Teile des Bezirks Aubonne. Einen wirtschaftlichen Aufschwung erlebte Morges im 17. und 18. Jahrhundert als wichtiger Handelsplatz und Standort der Berner Genferseeflotte.
+
+Nach dem Zusammenbruch des Ancien régime gehörte die Stadt von 1798 bis 1803 während der Helvetik zum Kanton Léman, der anschliessend mit der Inkraftsetzung der Mediationsverfassung im Kanton Waadt aufging. 1798 wurde die Stadt Hauptort des Bezirks Morges. Ein weiterer Aufschwung erfolgte im Lauf des 19. Jahrhunderts mit dem Anschluss an das schweizerische Eisenbahnnetz.
+
+Am 29. April 1961 wurde in Morges das Manifest von Morges (engl. Morges Manifesto) unterzeichnet, das zur Gründung des WWF führte.[5] Auch das erste Büro des WWF wurde in Morges eröffnet (11. September 1961).[6]
+
+## Sehenswürdigkeiten
+
+Parc de l'Indépendance - ideal für romantische längere Spaziergänge.  
+![](/Europa/Schweiz/morges/park.png)  
+
+![Sehenswürdigkeiten der Region](https://www.morges-tourisme.ch/de/GP1824/die-unumgaenglichen-sehenswuerdigkeiten)
+
+## Tipps
+
+Kultur und Natur - so kompakt wie sonst nirgends. Wandern oder biken und am Abend einer der vielen Anlässe in der Region besuchen.  
+
+Der [Weisswein](https://www.morges-tourisme.ch/de/GP1822/die-winzer) und die Gerichte der Waadt überzeugen auch den anspruchsvollsten Geniesser.  
+
+## Bild  
+
+![Morges](/Europa/Schweiz/morges/Morges_Switzerland.png)  
```

Comment:
Her wird mir kein Bild angezeigt. 
Die Dateiendung müsste .jpg sein

Reply:
1. Code von PNG auf JPG geändert
2. Mein Fork lets_git_reisefuehrer ist nicht aktuell. D.h. entspricht nicht diesem Repository.
Wie kann ich es "rebasen"?
3. Branch "StadtThun" ist lokal noch da. In diesem (Orginal) Repository gemerged. Wie update ich das lokale Repository? Auch in meinem Fork wird der Branch "StadtThun" angezeigt.
4. Branch "Stadt Morges" (dieser) angefangen, bevor Branch "Stadt Thun" in diesem Repository zum Master hinzugefügt wurde. Wie zeige ich den Inhalt (LS / DIR) eines Branches an? So das ich sehe, ob Thun dabei ist?
5. "rm -r europa\schweiz\thun" zeigt einen Fehler an